rundeckapp/src/test/groovy/rundeck/ScheduledExecutionTest.groovy

```diff
@@ -481,12 +481,12 @@ class ScheduledExecutionTest  {
 
     @Test void testGenerateJobGroupName() {
         def ScheduledExecution se = new ScheduledExecution()
-        se.properties = [jobName: 'TestName', project: "AFrameworkProject"]
-        assertEquals "incorrect group name: ${se.generateJobGroupName()}", "AFrameworkProject:TestName:", se.generateJobGroupName()
+        se.properties = [uuid: "a1", jobName: 'TestName', project: "AFrameworkProject"]
+        assertEquals "incorrect group name: ${se.generateJobGroupName()}", "AFrameworkProject", se.generateJobGroupName()
 
         se = new ScheduledExecution()
-        se.properties = [jobName: 'TestName', project: "AFrameworkProject", groupPath: 'The Group']
-        assertEquals "incorrect group name: ${se.generateJobGroupName()}", "AFrameworkProject:TestName:The Group", se.generateJobGroupName()
+        se.properties = [uuid: "a1", jobName: 'TestName', project: "AFrameworkProject", groupPath: 'The Group']
+        assertEquals "incorrect group name: ${se.generateJobGroupName()}", "AFrameworkProject", se.generateJobGroupName()
```

Comment:
comment: FYI: since we want to try to have everything in Spock, I try to avoid modifying existing Junit tests and just convert them to Spock equivalents when I have to. No need to do that now but for future PRs we can try to make that change.